types/dynamodb-lock-client/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+      "module": "commonjs",
+      "lib": ["es6"],
+      "noImplicitAny": true,
+      "noImplicitThis": true,
+      "strictNullChecks": true,
+      "baseUrl": "../",
+      "typeRoots": ["../"],
+      "types": [],
+      "noEmit": true,
+      "forceConsistentCasingInFileNames": true,
+      "esModuleInterop": true,
```

Comment:
Remove `esModuleInterop` please, to prevent mistakes in future updates.

Reply:
Will do, IIRC I added this because VSCode was having issues.